tests/xbot/common/properties/PropertyTest.java

```diff
@@ -65,6 +66,57 @@ public void testSavingValue() {
     }
     
     @Test
+    public void testSavingValuePersistent() {
+        DoubleProperty dbl1 = propertyManager.createProperty("speed", 0.5);
+        BooleanProperty bool1 = propertyManager.createProperty("isTrue", false);
+        StringProperty str1 = propertyManager.createProperty("string", "test2");
+        
+        DoubleProperty dbl2 = propertyManager.createProperty("weight", 2.3, PropertyPersistenceType.Ephemeral);
+        BooleanProperty bool2 = propertyManager.createProperty("isFalse", true, PropertyPersistenceType.Ephemeral);
+        StringProperty str2 = propertyManager.createProperty("robotname", "xbot", PropertyPersistenceType.Ephemeral);
+        
+        DoubleProperty dbl3 = propertyManager.createProperty("height", 4.8, PropertyPersistenceType.Persistent);
+        BooleanProperty bool3 = propertyManager.createProperty("isAwesome", true, PropertyPersistenceType.Persistent);
+        StringProperty str3 = propertyManager.createProperty("team", "488", PropertyPersistenceType.Persistent);
+        
+        assertSame(null, propertyManager.permanentStore.getDouble("speed"));
+        assertSame(null, propertyManager.permanentStore.getBoolean("isTrue"));
+        assertSame(null, propertyManager.permanentStore.getString("string"));
+        
+        assertSame(null, propertyManager.permanentStore.getDouble("weight"));
+        assertSame(null, propertyManager.permanentStore.getBoolean("isFalse"));
+        assertSame(null, propertyManager.permanentStore.getString("robotname"));
+        
+        assertSame(null, propertyManager.permanentStore.getDouble("height"));
+        assertSame(null, propertyManager.permanentStore.getBoolean("isAwesome"));
+        assertSame(null, propertyManager.permanentStore.getString("team"));
+        
+        assertEquals(0.5, dbl1.get(), 0.001);
```

Comment:
To be honest I didn't even gave a thought about this, I just copied over the tests from the normal save, and it was using that margin of error. Maybe @JohnGilb or @aschokking can comment on why we use a 0.001 margin of error.